### PR TITLE
add Home Work Js1w1

### PR DESCRIPTION
**Pull request by mehmetekinci**

### CODE REVIEW

File: script2.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*console.log('I'm awesome');
+
+Correction */
+
+console.log('I\'m awesome');
+console.log("I'm awesome");
+console.log(`I'm awesome`);
```
Comment (nudge) [inline script2.js:7]: 👍

File: script3.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+x = 5;
+
+/*3.5*/
+
+console.log("the value of my variable x will be: 5");
+
+/*3.6*/
+
+console.log(x);
```
Comment (nudge) [inline script3.js:23]: 👍

File: script4.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+y = 'Hello World!';
+
+/* 4.4 */
+
+console.log("the value of my variable y will be: Hello World!");
+
+/* 4.5 */
+
+console.log(y);
```
Comment (nudge) [inline script4.js:24]: 👍

File: script5.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+/* 5.3 */
+
+let a = Math.round(z);
+
+/* 5.4 */
+
+console.log(a);
+
+/* 5.5 */
+let Biggest = Math.max(a, z);
```
Comment (nudge) [inline script5.js:19]: Good use of `Math.max()`!

File: script5.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/* 5.1 */
+
+let z;
+z = 7.25;
```
Comment (nudge) [inline script5.js]: You could also do `let z = 7.25;` in one line

File: script6.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+/* 6.1 */
+
+let Cars = [];
```
Comment (nudge) [inline script6.js]: It is convention to use camelCase to name variables, so this could be named `cars`

File: script6.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+/* 6.2 */
+
+console.log("the value of my variable y will be: []");
```
Comment (nudge) [inline script6.js]: nit: small change, variable is not named `y`, should be `cars`

File: script6.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+console.log(myFavoriteAnimals);
+
+/* 6.6 */
+
+myFavoriteAnimals.push('baby_pig');
+
+/* 6.7 */
+
+console.log(myFavoriteAnimals);
```
Comment (nudge) [inline script6.js]: :+1: good use of camelCase here to name the variable!

File: script7.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+console.log(myString);
+
+/* 7.2 */
+
+let n = myString.length;
+
+/* 7.3 */
+
+console.log(n);
```
Comment (nudge) [inline script7.js:15]: :+1:

**Comment by nudge** [comment]
Very nice work! :)

File: script2.js (hunk 1 above)
Comment (mehmetekinci) [reply script2.js:7]: thanks :)

File: script3.js (hunk 2 above)
Comment (mehmetekinci) [reply script3.js:23]: thanks :)

File: script4.js (hunk 3 above)
Comment (mehmetekinci) [reply script4.js:24]: thanks :)

File: script5.js (hunk 4 above)
Comment (mehmetekinci) [reply script5.js:19]: thanks :)

File: script5.js (hunk 5 above)
Comment (mehmetekinci) [reply script5.js]: Thanks I get it. I will correct.

File: script6.js (hunk 6 above)
Comment (mehmetekinci) [reply script6.js]: Thanks I get it. I will correct.

File: script6.js (hunk 7 above)
Comment (mehmetekinci) [reply script6.js]: Thanks I will correct.

File: script6.js (hunk 8 above)
Comment (mehmetekinci) [reply script6.js]: thanks :)

File: script7.js (hunk 9 above)
Comment (mehmetekinci) [reply script7.js:15]: thanks :)

**Pull request closed without merging**